src/salesforce_uploader.js

```diff
@@ -35,29 +37,44 @@ type apiGatewayLambdaInput = {
   headers: inputHeaders
 
 }
+
+async function copyToUploadedFolder (stage, s3Path, sfFileName) {
+  try {
+    let uploadedPath = `${stage}/uploaded/${sfFileName}`
+    await
+      copyObject(s3Path, uploadedPath)
+  } catch (err) {
+    console.error('error copying fulfilment file to uploaded directory')
+    console.log(err)
```

Comment:
Unless s3 throws includes the contents of the file as part of the exception it should be safe